### PR TITLE
Fix/tokens display

### LINKED ISSUE #2812: Grey out expired tokens in FlowAuth
The list of previously-created tokens in FlowAuth includes all tokens (including those that have expired). The expiry is displayed, but it would be useful to either grey out or not show the expired tokens (so that a user doesn't copy/download an expired token by mistake).

### PR DESCRIPTION
**Pull request by flowstef**
Closes #2812

### I have:

- [x] Brought the branch up to date with main
- [x] Added any relevant Github labels
- [ ] Added tests for any new additions (it doesn't look like we've got any JS tests yet)
- [x] Added or updated any relevant documentation
- [x] Updated the [Changelog](https://github.com/Flowminder/FlowKit/blob/master/CHANGELOG.md) 

### Description

Adds visual indications of expired tokens and sorts tokens to display the longest lasting ones first to avoid excessive scrolling.
Also related to #1124 I think.

### CODE REVIEW

**Comment by greenape** [comment]
Looks like this:

Screenshot 2021-02-09 at 11 01 49

**Review by greenape — CHANGES_REQUESTED** [review]
Nice :)

Two tiny things, otherwise great.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Added
+- Improvements to token display in FlowAuth
```
[the remark this answers is not in the document]
Comment (flowstef) [reply CHANGELOG.md:10]: Sure! Looks like the change has already been made. It's also been linked when I tagged the issue in this PR - is that what you meant? If so I guess I can resolve the conversation.

[the remark this answers is not in the document]
Comment (greenape) [reply CHANGELOG.md:10]: I was thinking move this entry to the Changed section and add `[#2812](https://github.com/Flowminder/FlowKit/issues/2812)` at the end :)

[the remark this answers is not in the document]
Comment (greenape) [reply CHANGELOG.md:10]: (GH will autoclose #2812 when this merges)

[the remark this answers is not in the document]
Comment (flowstef) [reply CHANGELOG.md]: Ah OK, I get you now. I didn't even see that I had duplicated the heading, thanks.

**Review by greenape — APPROVED** [review]
Awesome, thanks! :)

**Pull request merged**